docs/sources/trackers/web-trackers/cross-domain-tracking/index.md

```diff
@@ -4,23 +4,239 @@ date: "2022-08-30"
 sidebar_position: 2850
 ---
 
-# Cross-domain tracking
-
 ```mdx-code-block
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
-import CrossDomain from "@site/docs/reusable/javascript-tracker-cross-domain/_index.md"
 ```
 
+Configure [cross-domain tracking](/docs/events/cross-navigation/index.md) using the `crossDomainLinker` and `useExtendedCrossDomainLinker` fields of the [configuration object](/docs/sources/trackers/web-trackers/tracker-setup/initialization-options/index.md).
+
+:::tip
+
+If you enable link decoration, aim to track at least one event on the starting page. The tracker writes the `domain_userid` to a cookie when it tracks an event. If the cookie doesn't exist when the user navigates to the cross-domain destination, the tracker will generate a new ID for them when they return, rather than keeping the old ID. This can make user stitching difficult.
```

Comment:
I don't believe this is true anymore? Just creating a tracker (required to set these settings) should be enough to create the cookie in question.